pages/about/index.tsx

```diff
@@ -183,160 +186,187 @@ const About: NextPage<{ teams: TeamSection[] }> = ({ teams }) => {
     <>
       <SEO title={t('seo.about_page_title')} description={t('seo.about_page_description')} />
       <div className="max-w-7xl m-auto pb-6">
-        <h1 className="text-primary-blue">
-          <Trans className="block my-2">{t('about_page.title')}</Trans>
-        </h1>
-        <H2>{t('about_page.movapp_goal_title')}</H2>
-        {t(`about_page.movapp_goal_description.${getCountryVariant()}`)}
-        <H2>{t('about_page.why_movapp_title')}</H2>
-        <P>
-          <Trans i18nKey={'about_page.why_movapp_mova'} />
-        </P>
-        <P>{t('about_page.why_movapp_description')}</P>
-        <P>
-          <Trans
-            i18nKey={'about_page.why_movapp_license'}
-            t={t}
-            components={[
-              <TextLink
-                href={`https://creativecommons.org/licenses/by-nc/4.0/deed.${currentLanguage}`}
-                target="_blank"
-                key="creativecommons"
-              />,
-            ]}
-          />
-        </P>
-
-        <Trans
-          i18nKey={'about_page.why_movapp_origin'}
-          t={t}
-          components={[
-            <TextLink
-              href="https://drive.google.com/drive/u/0/folders/129vObZ0vUHpDd07slIfaiAfKsEbx1mNw"
-              target="_blank"
-              key="drive.google.com"
-            />,
-          ]}
-        />
-
-        {currentLanguage.toString() === 'cs' || currentLanguage.toString() === 'uk' ? (
-          <>
-            <H2>{t('about_page.movapp_origin_title')}</H2>
+        <div className="max-w-[780px] text-center mx-auto py-[124px]">
+          <h1 className="text-primary-blue whitespace-pre-line ">
+            <Trans className="block my-2">{t('about_page.title')}</Trans>
+          </h1>
+        </div>
+        {/* section main__________________________________________________________________*** */}
+        <section className=" columns-2 gap-[55px] auto-cols-min px-4 md:text-sm">
```

Comment:
Nechybí v className `grid`, pokud používáš `auto-cols-min`?

Reply:
Sloupce (pro větší obrazovku) můžeš vytvovřit pomocí `grid md:grid-flow-col`. pak umístit poslední div, why_movapp, do 2. sloupce podle designu. `className="row-span-2 ...`.

V tomto případě se nehodí `auto-cols-min`, nastaví šířku sloupců dle nejdelšího slova.